README.md

```diff
@@ -58,8 +58,21 @@ Technical meetings on [KERISSE](kerisse.org) take place twice a week. Drop us a
 The repo for the Web of Trust terminology and glossaries is [here](https://github.com/WebOfTrust/WOT-terms). 
 
 ## Existing Implementations
-The reference implementation for KERI is the keripy implementation. This is hosted here by the WebOfTrust project. The repository may be found here:  
-* [keripy](https://github.com/WebOfTrust/keripy) – under development  
+The reference implementation for KERI is the keripy (Python) implementation. This is hosted here by the WebOfTrust project. See the below table for where the reference implementation, and others, are located.
+
+| Component               | Name                                                       | Language   | Creators                      | Status                        | Current Version                                                        | Spec Compliance |
```

Comment:
Would suggest to remove version from that table because:
- it does not tell you anything
- it would be obsolete very quickly

Reply:
It does give a rough indication of the level of development of a repository. Although it would be obsolete quickly it could be refreshed quarterly or so just to help keep people informed at a glance on what is happening in the space.

Reply:
I changed it to say "Recent Version" as a middle ground.